### PR TITLE
Changes to use existing sqlServer in builds

### PR DESCRIPTION
**Pull request by rbans96**
## Description
Describe the changes in this PR.

## Related issues
Addresses [issue #].

## Testing
Describe how this change was tested.

## FHIR Team Checklist
- [ ] **Update the title** of the PR to be succinct and less than 50 characters
- [ ] **Add a milestone** to the PR for the sprint that it is merged (i.e. add S47)
- [ ] Tag the PR with the type of update: **Bug**, **Dependencies**, **Enhancement**, or **New-Feature**
- [ ] Tag the PR with **Azure API for FHIR** if this will release to the Azure API for FHIR managed service (CosmosDB or common code related to service)
- [ ] Tag the PR with **Azure Healthcare APIs** if this will release to the Azure Healthcare APIs managed service (Sql server or common code related to service)
- [ ] CI is green before merge [Build Status](https://microsofthealthoss.visualstudio.com/FhirServer/_build/latest?definitionId=27&branchName=main) 
- Review [squash-merge requirements](https://github.com/microsoft/fhir-server/blob/master/SquashMergeRequirements.md)

### Semver Change ([docs](https://github.com/microsoft/fhir-server/blob/master/docs/Versioning.md))
Patch|Skip|Feature|Breaking (reason)

### CODE REVIEW

File: build/jobs/provision-deploy.yml [hunk 1]
```diff
@@ -22,6 +22,12 @@ parameters:
 - name: schemaAutomaticUpdatesEnabled
   type: string
   default: 'tool'
+- name: sqlServerAdminPassword
+  type: string
+  default: 'test@123'
```
Comment (LTA-Thinking) [inline build/jobs/provision-deploy.yml:27]: I don't think we should provide a default password for the SQL server.

Comment (LTA-Thinking) [inline build/jobs/provision-deploy.yml:27]: Could we remove the default value for this?

File: build/jobs/provision-sqlServer.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+- name: resourceGroup
+  type: string
+- name: sqlServerName
+  type: string
+- name: schemaAutomaticUpdatesEnabled
+  type: string
+  default: 'auto'
+- name: sqlServerAdminPassword
+  type: string
+  default: 'test@123'
```
Comment (LTA-Thinking) [inline build/jobs/provision-sqlServer.yml:12]: Could we remove this default value?

File: build/jobs/provision-deploy.yml (hunk 1 above)
Comment (rbans96) [reply build/jobs/provision-deploy.yml]: Ah missed, done!

Comment (rbans96) [reply build/jobs/provision-deploy.yml]: Just realized that we would need to set default value as provision-deploy.yml is used for cosmosdb site as well. I am setting it to empty value as default which will error out if we don't pass in good value for sql Server

**Review by LTA-Thinking — APPROVED** [review]

**Pull request merged**